src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/reports/ImportReportsService.kt

```diff
@@ -44,9 +44,17 @@ class ImportReportsService(
     failIfDateOfImportNotInPast(date)
     failIfAnyReportFilesAreMissingOn(date)
 
-    importMovesJourneysEventsOn(date)
-    importPeopleOn(date)
-    importProfilesOn(date)
+    val maybeMovesAuditableEvent = importMovesJourneysEventsOn(date)
+    val maybePeopleAuditableEvent = importPeopleOn(date)
+    val maybeProfilesAuditableEvent = importProfilesOn(date)
+
+    if (maybeMovesAuditableEvent != null && maybePeopleAuditableEvent != null && maybeProfilesAuditableEvent != null) {
```

Comment:
An option here would be to put this into one event instead of 3 which may be cleaner.